pkg/asset/ignition/bootstrap/bootstrap.go

```diff
@@ -31,7 +31,7 @@ import (
 const (
 	rootDir              = "/opt/openshift"
 	bootstrapIgnFilename = "bootstrap.ign"
-	etcdCertSignerImage  = "quay.io/coreos/kube-etcd-signer-server:678cc8e6841e2121ebfdb6e2db568fce290b67d6"
+	etcdCertSignerImage  = "registry.svc.ci.openshift.org/openshift/origin-v4.0:kube-etcd-signer-server"
```

Comment:
Why this?  Can we drop this commit until we move the etcd signer into the release image references?

Reply:
> Why this? Can we drop this commit until we move the etcd signer into the release image references?

Sure I just figured it was better to not have the non RHEL image, will update.

Reply:
Hmm, now that you've dropped this, is it still fixing rhbz#1684670 ?  I think we need openshift/origin#22325 so we can drop `etcdCertSignerImage` completely.

Reply:
https://github.com/openshift/origin/pull/22325#issuecomment-473357594

Reply:
So if that if signer-server is refed in origin we can just pull the sha like the others or?

`podman run --quiet --rm ${release} image kube-etcd-signer-server`

Wouldn't this work now?

Reply:
guess not

```
podman run --quiet --rm registry.svc.ci.openshift.org/ci-op-mnr8nyp6/release:latest image kube-etcd-signer-server
[...]
 1 image.go:36] error: error: Unknown name requested, could not find kube-etcd-signer-server in UpdatePayload
```

Reply:
> So if that if signer-server is refed in origin we can just pull the sha like the others or?
> 
> `podman run --quiet --rm ${release} image kube-etcd-signer-server`
> 
> Wouldn't this work now?

only images required by operators and some hard coded defaults are available in the release image.
[from this CI run release image]
```console
oc adm release info registry.svc.ci.openshift.org/ci-op-mnr8nyp6/release:latest --image-for=kube-etcd-signer-server                                                                                                                              error: no image tag "kube-etcd-signer-server" exists in the release image registry.svc.ci.openshift.org/ci-op-mnr8nyp6/release:latest
```

Reply:
> images required by operators

this is defined by `image-references` file [example](https://github.com/openshift/machine-config-operator/blob/master/install/image-references) for each openshift operators

> hard coded defaults

this is defined [here](https://github.com/openshift/origin/blob/master/pkg/oc/cli/admin/release/new.go#L49)

Reply:
I see so with openshift/origin#22325 updated per your comment we will want

`ETCD_CERT_SIGNER_IMAGE=$(podman run --quiet --rm ${release} image kube-etcd-signer-server)` ? and remove `etcdCertSignerImage` ref?

Reply:
added https://github.com/openshift/cluster-bootstrap/pull/20